### ISSUE TITLE
Add tailwind as an option

### ISSUE THREAD

**Issue by medaminefh** [vuejs/create-vue#246]
This is handled here #245

**Comment by iampapagray** [comment]
Any update on this?

**Comment by medaminefh** [comment]
@iampapagray  No no updates here from my side, there is a problem with picking that option (Tailwind) along with Typescript as I mentioned here #245, but it works without Typescript(I've test it before)

**Comment by iampapagray** [comment]
@medaminefh Right. I created a new project with Typescript option. And added Tailwind using the steps here: https://tailwindcss.com/docs/guides/vite#vue . It works fine for now.
